### DISCUSSION TITLE
Reset Password Mail received without any link for resetting the password

### DISCUSSION THREAD

**Discussion by ashab96** [supabase/supabase#15492] [Questions]
I have setup the nextjs +supbase Auth App.
1) I  am using Supabase Auth UI , Signin and Sign up are successful 
2)But when I verify the forgot password functionality , I am facing a issue .
3)On Clicking the forgot password link , I am successfully able to get reset password mail from supabase , but the mail does not have link to reset the password 

could anyone help me how can I fix this issue .
![supabase_mail](https://github.com/supabase/supabase/assets/28101438/98aebd41-966e-4462-9c96-81a227f08bc3)
![ui](https://github.com/supabase/supabase/assets/28101438/57d98f43-0d85-4bca-86be-4a1690d132c1)
<img width="894" alt="image" src="https://github.com/supabase/supabase/assets/28101438/a08e8793-da52-40d2-a3d1-d3da3dc2819c">

**Comment by Nathx** [comment]
Hi ashab, marking the link as safe in Gmail displays the Reset password link, this is a safety feature from Google.
